src/main/java/uk/gov/justice/digital/delius/controller/secure/ReferralController.java

```diff
@@ -35,9 +41,9 @@
                 @ApiResponse(code = 500, message = "Unrecoverable error whilst processing request.", response = ErrorResponse.class)
             })
 
-    @ApiOperation(value = "Adds a sent referral contact entry to the contact log")
-    public ResponseEntity<String> createReferralSent(final @PathVariable("crn") String crn,
-                                                     final @RequestBody @Valid ReferralSentRequest referralSentRequest) {
-        return referralService.createReferralSent(crn, referralSentRequest);
+    @ApiOperation(value = "Creates an NSI referral")
+    public ResponseEntity<ReferralSentResponse> createReferralSent(final @PathVariable("crn") String crn,
+                                                                   final @RequestBody @Valid ReferralSentRequest referralSentRequest) {
+        return new ResponseEntity<>(referralService.createNsiReferral(crn, referralSentRequest), HttpStatus.OK);
```

Comment:
you could change this to `return neweferralService.createNsiReferral(crn, referralSentRequest)`

No need for OK since that is the default